core/trino-main/src/main/java/io/trino/operator/MultiChannelGroupByHash.java

```diff
@@ -684,9 +684,13 @@ public Void getResult()
     class AddLowCardinalityDictionaryPageWork
             extends LowCardinalityDictionaryWork<Void>
     {
+        private final int[] combinationIdToPosition;
+        private int nextCombinationIndex;
```

Comment:
`nextCombinationIndex` -> `nextCombinationId`

Reply:
That kind of makes it sound like the value is the `combinationId` when in reality it's the index of the next combinationId. I'm fine with changing it if you insist, but I avoided that name initially to try to make it more clear.